src/i18n.ts

```diff
@@ -0,0 +1,47 @@
+import i18next from "i18next";
+import LanguageDetector from "i18next-browser-languagedetector";
+import { initReactI18next } from "react-i18next";
+
+i18next
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    debug: true,
+    fallbackLng: "en",
+    lng: "en-GB",
+    resources: {
+      en: {
+        translation: {
+          name: "Name",
+          email: "Email",
+          type: "Choose type",
+          complaint: "Complaint",
+          submit: "Submit",
+          title: "Complaint Box!",
+          subTitle: "Get assured resolution within 24 hours",
+          tnc: "Excluding weekends and public holidays",
+          tickets: "Active tickets",
+          intlDate: "Date:{{createdAt, IntlDate}}",
```

Comment:
All other keys have the variable name as `val` why does this have `createdAt`?

Reply:
Modified it.